src/plugins/intel_npu/src/al/include/intel_npu/al/config/common.hpp

```diff
@@ -142,7 +142,11 @@ struct LOG_LEVEL final : OptionBase<LOG_LEVEL, ov::log::Level> {
     }
 
     static ov::log::Level defaultValue() {
+#if defined(NPU_PLUGIN_DEVELOPER_BUILD) || !defined(NDEBUG)
+        return ov::log::Level::WARNING;
+#else
         return ov::log::Level::NO;
+#endif
```

Comment:
this is to fix issue.
when no log_level env is set, and build project with release version and enable `NPU_PLUGIN_DEVELOPER_BUILD`, Log_lovel parsed by  `_globalConfig.parseEnvVars()` is going to be the default value 'ov::log::Level::NO', which is  inconsistent with the `Logger::global()`